manuals/CentOS-7-8-source.en.md

```diff
@@ -1,5 +1,12 @@
 # CentOS 7 and 8 - Source Installation
 
+IMPORTANT:
+* As of Jun 30, 2024 CentOS 7 has reached EOL.
+* As of May 31, 2024 CentOS Stream 8 has reached EOL.
+
+This instruction remains for historical purposes. Please find the currently maintained
+instruction for the software probe [here](Linux-source.en.md).
```

Comment:
do not use `here` for links as `here` does not contain information about the destination.

Suggestion: *Please follow the currently maintained [instruction for the software probe](Linux-source.en.md).*

Reply:
I will close this merge request and send an updated one.